cumulusci/tasks/bulkdata.py

```diff
@@ -139,6 +141,7 @@ class LoadData(BaseSalesforceApiTask):
     def _run_task(self):
         self._init_mapping()
         self._init_db()
+        self.logger.debug('here')
```

Comment:
Should this be here?

Reply:
nope. and was gone at one point but merged its way back in...